File: observer/Samuel Sales/DesafioObserver_Samuel/src/Canais/Globo.java
package Canais;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class Globo implements ObserverGrupoGlobo {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Canal Globo");
    }
}

File: observer/Renny/PadraoObserverII/Canais/Globo.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Canais
{
    class Globo : IObservar
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Canal Globo!");
        }
    }
}
